logviewer cause index out of range when mpijob is not ready

submit mpijob then use `arena logviewer` cause panic
```
arena logviewer mpi
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/kubeflow/arena/cmd/arena/commands.(*MPIJob).GetJobDashboards(0xc420655500, 0xc42028c000, 0x10, 0x11cec8b, 0x7, 0x12e75e0, 0xc420655500)
	/go/src/github.com/kubeflow/arena/cmd/arena/commands/trainer_mpi.go:143 +0x4b5
github.com/kubeflow/arena/cmd/arena/commands.NewLogViewerCommand.func1(0xc4203d0c80, 0xc420294aa0, 0x1, 0x1)
	/go/src/github.com/kubeflow/arena/cmd/arena/commands/logviewer.go:55 +0x241
github.com/kubeflow/arena/vendor/github.com/spf13/cobra.(*Command).execute(0xc4203d0c80, 0xc420294a50, 0x1, 0x1, 0xc4203d0c80, 0xc420294a50)
	/go/src/github.com/kubeflow/arena/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/kubeflow/arena/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420286a00, 0xc4203d0c80, 0xc4203d0a00, 0xc4203d0780)
	/go/src/github.com/kubeflow/arena/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/kubeflow/arena/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420286a00, 0x405a9c, 0xc4200a8058)
	/go/src/github.com/kubeflow/arena/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/go/src/github.com/kubeflow/arena/cmd/arena/main.go:38 +0x38
```

--- a/cmd/arena/commands/trainer_mpi.go
+++ b/cmd/arena/commands/trainer_mpi.go
@@ -136,6 +136,10 @@ func (mj *MPIJob) GetJobDashboards(client *kubernetes.Clientset) ([]string, erro
 		return urls, fmt.Errorf("No LOGVIEWER Installed.")
 	}
 
+	if len(mj.chiefPod.Spec.Containers) == 0 {
+		return urls, fmt.Errorf("mpi launcher is not ready!")
+	}
+
 	url := fmt.Sprintf("%s/#!/log/%s/%s/%s?namespace=%s\n",
 		dashboardURL,
 		mj.chiefPod.Namespace,

